src/App.js

```diff
@@ -14,11 +15,68 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    good: 'Jam',
+  }
+
+  render() {
+    const { good } = this.state;
+
+    return (
+      <div className="goods">
+        {good ? (
+          <h1 className="goods__title">
+            {good}
+            {' '}
+            is selected
+            {' '}
+            <button
+              type="button"
+              onClick={() => {
+                this.setState({ good: null });
+              }}
+            >
+              X
+            </button>
+          </h1>
+        ) : (
+          <h1 className="goods__title">
+            No goods selected
+          </h1>
+        )
+        }
+        <ul className="goods__list">
+          {goodsFromServer.map(goodFS => (
+            <li
+              key="goodFS"
```

Comment:
```suggestion
              key={goodFS}
```
Now you're giving for the `key` attribute literally the value `'goodFS'` (string). Al your `li` will have the same key - string 'goodFS'. You shoul use the value of the variable `goodFS`, so it shoul be written i parentheses